skyportal/handlers/api/spectrum.py

```diff
@@ -149,7 +149,7 @@ def post(self):
 
         self.push_all(
             action='skyportal/REFRESH_SOURCE_SPECTRA',
-            payload={'obj_key': spec.obj.internal_key},
+            payload={'obj_id': spec.obj.id},
```

Comment:
??

Reply:
Just a small bug I found along the way.

Reply:
So the front-end actually checks obj_id and not obj_key?

Reply:
The handler for REFRESH_SOURCE_SPECTRA does. Maybe that is also a bug?

Reply:
So if it is a bug, can you please fix it, too? Or make sure it's not a bug. There was an issue when uploading a spectrum, it wouldn't appear on the source page unless you reload the page, but @acrellin fixed it recently.

Reply:
This is the bug fix. The only thing is that I'm not sure why we are using internal key and not just the object id in all the refresh messages.

Reply:
We were using the internal key because, for a time, object IDs were considered sensitive, access-restricted information. But since objects became read-public, some message handlers use `obj_id` while others still use `internal_key`. I confirmed that the `spectra` message handler does indeed expect `obj_id`, so this looks good to me. We should bring all of these into consistency soon, while I'll file an issue for (perhaps involving dropping `Obj.internal_key`, as that's no longer relevant).

Reply:
Great. I was hoping I wouldn't need to go replace all the object ids with internal keys.